internal/datanode/services.go

```diff
@@ -334,7 +342,7 @@ func (node *DataNode) SyncSegments(ctx context.Context, req *datapb.SyncSegments
 						log.Warn("failed to DecompressBinLog", zap.Error(err))
 						return val, err
 					}
-					pks, err := util.LoadStats(ctx, node.chunkManager, ds.metacache.Schema(), newSeg.GetSegmentId(), []*datapb.FieldBinlog{newSeg.GetPkStatsLog()})
+					pks, err := compaction.LoadStats(ctx, node.chunkManager, ds.GetMetaCache().Schema(), newSeg.GetSegmentId(), []*datapb.FieldBinlog{newSeg.GetPkStatsLog()})
```

Comment:
Doesn't really make sense.
What's run with util.LoadStats? We can also load Stats if it's not under compaction